Makefile

```diff
@@ -1,11 +1,11 @@
 export PYTHONPATH=$(shell pwd)/hyperon_das_atomdb/
 export PYTHONDONTWRITEBYTECODE=1
 
-test:
```

Comment:
It seems like you removed the `test` target.

Reply:
Yes. Now we have `test-integration` to run only integration tests and `test-coverage` (We already had this target which is better than Test because it shows us coverage in the terminal).

The `test` and `test-coverage` commands did the same thing, but coverage showed coverage and the other didn't. So, I thought I'd better remove the other one

Reply:
ok but so please rename it to test-unit